LanternExtractor/EQ/Wld/Exporters/GltfWriter.cs

```diff
@@ -1020,7 +1022,7 @@ private IMeshBuilder<MaterialBuilder> InstantiateMeshBuilder(string meshName, bo
         }
 
         private void AddAnimatedMeshMorphTargets(Mesh mesh, IMeshBuilder<MaterialBuilder> gltfMesh,
-            string meshName, Matrix4x4 transformMatrix, Dictionary<VertexPositionNormal, int> gltfVertexPositionToWldVertexIndex)
+            string meshName, Matrix4x4 transformMatrix, Dictionary<VertexPositionNormal, int> gltfVertexPositionToWldVertexIndex, bool mirrorAxis)
```

Comment:
Minor - mirrorXAxis instead of mirrorAxis

Reply:
Fixed